File: Java/Mahasiswa.java
// membuat kelas mahasiswa
public class Mahasiswa{
    // atribut yang diperlukan
    private String nama;
    private int nim;
    private String prodi;
    private String fakultas;
    
    // konstruktor kosong
    Mahasiswa() {
    }

    // konstruktor dengan parameter
    Mahasiswa(String nama, int nim, String prodi, String fakultas) {
        this.nama = nama;
        this.nim = nim;
        this.prodi = prodi;
        this.fakultas = fakultas;
    }

    // tetapkan nama
    public void setNama(String nama) {
        this.nama = nama;
    }

    // get nama
    public String getNama() {
        return this.nama;
    }
    
    // tetapkan nim
    public void setNim(int nim) {
        this.nim = nim;
    }

    // get nim
    public int getNim() {
        return this.nim;
    }

    // tetapkan prodi
    public void setProdi(String prodi) {
        this.prodi = prodi;
    }

    // get prodi
    public String getProdi() {
        return this.prodi;
    }

    // tetapkan fakultas
    public void setFakultas(String fakultas) {
        this.fakultas = fakultas;
    }
    
    // get fakultas
    public String getFakultas() {
        return this.fakultas;
    }
}

File: CPP/Mahasiswa.cpp
#include <iostream>
#include <string>

using namespace std;

// membuat kelas mahasiswa
class Mahasiswa{
    // atribut yang diperlukan
    private:
        string nama;
        int nim;
        string prodi;
        string fakultas;
    
    public:
        // konstruktor kosong
        Mahasiswa() {
        }

        // konstruktor dengan parameter
        Mahasiswa(string nama, int nim, string prodi, string fakultas) {
            this->nama = nama;
            this->nim = nim;
            this->prodi = prodi;
            this->fakultas = fakultas;
        }

        // tetapkan nama
        void setNama(string nama) {
            this->nama = nama;
        }

        // get nama
        string getNama() {
            return this->nama;
        }

         // tetapkan nim
        void setNim(int nim) {
            this->nim = nim;
        }

        // get nim
        int getNim() {
            return this->nim;
        }

         // tetapkan prodi
        void setProdi(string prodi) {
            this->prodi = prodi;
        }

        // get prodi
        string getProdi() {
            return this->prodi;
        }

         // tetapkan fakultas
        void setFakultas(string fakultas) {
            this->fakultas = fakultas;
        }
        
        // get fakultas
        string getFakultas() {
            return this->fakultas;
        }

        // menampilkan data
        void tampilkan() {
            cout << "Nama     : " << nama << endl;
            cout << "Nim      : " << nim << endl;
            cout << "Prodi    : " << prodi << endl;
            cout << "Fakultas : " << fakultas << endl;
        }
        
        // deconstruct
        ~Mahasiswa() {
        }
};
